src/compile/scale/properties.ts

```diff
@@ -192,8 +192,8 @@ export function bins(model: Model, fieldDef: TypedFieldDef<string>, channel: Cha
   return undefined;
 }
 
-export function interpolate(channel: Channel) {
-  if (contains([COLOR, FILL, STROKE], channel)) {
+export function interpolate(channel: Channel, scaleType: ScaleType) {
+  if (contains([COLOR, FILL, STROKE], channel) && scaleType !== 'ordinal') {
```

Comment:
We might want interpolation for Vega-Lite `ordinal` but not `nominal` encodings. Let's see what changes.

Reply:
oh right

Reply:
Changed.